src/coreclr/gc/gc.cpp

```diff
@@ -45846,6 +45979,21 @@ unsigned int GCHeap::WhichGeneration (Object* object)
     return g;
 }
 
+enable_no_gc_region_callback_status GCHeap::EnableNoGCRegionCallback(uint64_t callback_threshold)
+{
+    return gc_heap::enable_no_gc_callback(callback_threshold);
+}
+
+int GCHeap::GetNoGCFinalizerAction(bool reset)
```

Comment:
I don't undertand why this is typed as `int` as opposed to the `enum` that was created.